crates/wysiwyg/src/composer_model/format.rs

```diff
@@ -449,4 +596,56 @@ mod test {
         model.format(InlineFormatType::Bold);
         assert_eq!(model.state.dom.to_string(), "<strong>hello world</strong>");
     }
+
+    #[test]
```

Comment:
I'd like to see a lot more tests in `test_formatting.rs` - then this looks good to go!